probatus/feature_elimination/feature_elimination.py

```diff
@@ -13,7 +13,7 @@
     preprocess_labels,
```

Comment:
For the failing unit test, you can modify the test to skip the check if the warning is there
`assert len(record) == 1`

Reply:
I've fixed this now